README.md

```diff
@@ -106,25 +106,30 @@ As it turns out, people usually find this document before they sign a contract w
 
 UPDATE after 2 years or so: nobody seems to actually track it, but now that we have more data, we know it takes approximately 90 minutes to set everything up, so in worst case we can compensate based on that value.
 
-### RT projects
```

Comment:
Maybe we could use RTI and RTO to make the naming consistent?

Reply:
Maybe we could. This document describes the current situation, which has been ongoing for >year, nobody cared about it enough to rename all projects and change the handbook. Personally I don't think it's worth the effort, but if you think it is, you can bring it up on S3. I think if you volounteer to go through the effort of making the changes, it'll pass, but if you'll try to make someone else do these changes, it might not.

For your consideration.